### ISSUE TITLE
Displays success if file not found

### ISSUE THREAD

**Issue by felix-last** [rasshofer/yaml-lint#3]
First of all thank you for this utility!
I think it's a bit confusing to see the output "√ YAML Lint successful." when the file specified on the CLI does not exist. Maybe the user should be warned instead?

**Example**
```
 » yamllint test.yaml
√ YAML Lint successful.
 » yamllint te.yaml
√ YAML Lint successful.
```
The result is the same even though the first file exists, but the second does not.

**Comment by rasshofer** [comment]
Ooops, thanks a lot for the hint! 🙈 This behavior is fixed now.
